docs/protos/kvetch/api/v1/api.proto

```diff
@@ -25,28 +22,25 @@ service API {
 
   // Subscribe will subscribe to a key or prefix and return the current value
   // and any changes.
-  rpc Subscribe(stream SubscribeRequest) returns (stream SubscribeResponse);
+  rpc Subscribe(SubscribeRequest) returns (stream SubscribeResponse);
 }
 
 message SetValuesRequest { repeated KeyValue messages = 1; }
 
 message SetValuesResponse {}
 
-message GetValuesRequest { repeated string keys = 1; }
+message GetValuesRequest {
+  // GetValue is a get value request.
+  message GetValue {
+    string key = 1;
+    bool is_prefix = 2;
+  }
+
+  repeated GetValue requests = 1;
+}
 
 message GetValuesResponse { repeated KeyValue messages = 1; }
 
-message SubscribeRequest {
-  oneof request {
-    Subscriptions request_subscriptions = 1;
-    CancelSubscription request_cancel_subscription = 2;
-    Ping request_ping = 3;
-  }
-}
+message SubscribeRequest { repeated string prefixes = 1; }
```

Comment:
does this need to be prefixes only? or perhaps a repeated of GetValue (maybe renamed something like KeySpecification)?

Reply:
Ah, didn't know that badger subscriptions were all prefixes anyway.  lg2m.

Reply:
So all subscriptions to badger are prefixes. Even if you do a single key you would have to do it as a prefix. I figured I would just push that out to the client instead of adding the code to ignore stuff in the service. I actually wrote the code to do both first